docs-src/docs.md

```diff
@@ -0,0 +1,422 @@
+
+![image alt text](images/image_11.png)
+
+Positive
+: Note: If Jenkins does not automatically start the job after a while click the clock icon to the right to trigger the job.
+
+After a short time you will see the job has failed and the **Last Failure** is set to a recent time. Click on the Job number **#2** (or any number after the hash sign) next to it to open the job details
+
+In the menu click **Console Output**. Here you can see the error that has happened with the build. Obviously someone did not close an if-statement properly and the linting process found that error early on.
+
+![image alt text](images/image_12.png)
+
+Now lets correct that error again and remove that mistake in our ConfigureTargetRequest.js file and maybe just add a comment to it.
+
+```
+vim apiproxy/resources/jsc/ConfigureTargetRequest.js
+```
+
+And again we commit our changes to the feature branch.
+
+```
+git commit -am "Improvement to ConfigureTargetRequest Script"
+git push origin feature/1
+```
+
+Switch back to Jenkins to see the status of the build process and after some time we should see the features branch with a blue circle and a **Last Success** date.
+
+![image alt text](images/image_13.png)
+
+In Apigee we now see the *currency-jenkinsv1* proxy deployed with the description of the proxy set to commit <commit_hash> from feature/1 branch by jenkins.
+
+![image alt text](images/image_14.png)
+
+**Merge to master (using the UI)**
+
+Now that the feature is complete, we create a pull request from feature branch to master.
+
+Positive
+: Note: Even though this is not a must-have but it is good to demonstrate a best-practice team development scenario where merge/pull requests are used for review/discussion of features before they are merged to master branch. If there are no conflicts, Github makes it particularly easy to merge from a pull request.
+
+In github open the **Pull requests** tab and create a **new pull request**
+
+![image alt text](images/image_15.png)
+
+Make sure to select the **master** as a base and your **feature/v1** branch as the branch to merge from. You can also see the commits we have done in that feature branch incl. our comments. Click **Create pull request** once you are done reviewing
+
+![image alt text](images/image_16.png)
+
+Now you can put some comments about your pull request and also have the option to put in reviewers or other attributes to that pull request. Click **Create pull request** to finish your pull request.
+
+![image alt text](images/image_17.png)
+
+Now that the pull request is created that's where it is a good practice to let some else approve your changes before we put the feature into the master branch. In that case we would switch hats to the lead developer and **Merge pull request** and click on **Confirm merge** after the review to put the changes into our master branch.
```

Comment:
lets add a comment first as part of a code review, and then merge

Reply:
put in a screenshot with a comment from the reviewer